controller_manager/test/test_load_controller.cpp

```diff
@@ -86,25 +87,34 @@ class TestLoadedController : public TestLoadController
   }
 
   void start_test_controller(
+    const int strictness,
     const std::future_status expected_future_status = std::future_status::timeout,
     const controller_interface::return_type expected_interface_status =
       controller_interface::return_type::OK)
   {
-    _switch_test_controllers(
-      strvec{controller_name1}, strvec{}, expected_future_status, expected_interface_status);
+    switch_test_controllers(
+      strvec{controller_name1}, strvec{}, strictness, expected_future_status,
+      expected_interface_status);
   }
 
   void stop_test_controller(
+    const int strictness,
     const std::future_status expected_future_status = std::future_status::timeout,
     const controller_interface::return_type expected_interface_status =
       controller_interface::return_type::OK)
   {
-    _switch_test_controllers(
-      strvec{}, strvec{controller_name1}, expected_future_status, expected_interface_status);
+    switch_test_controllers(
+      strvec{}, strvec{controller_name1}, strictness, expected_future_status,
+      expected_interface_status);
   }
 };
 
-TEST_F(TestLoadedController, load_and_configure_one_known_controller)
+class TestLoadedControllerParameterzied : public TestLoadedController,
```

Comment:
```suggestion
class TestLoadedControllerParametrized : public TestLoadedController,
```
sorry

also where is codespell?

Reply:
no problem, but codespell does not check the the code but comments